src/components/Card/Card.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import './card.css'
+
+function Card({ jobData }) {
+  let date = ''
+  try {
+    date = new Date(jobData.created).toLocaleDateString()
+  } catch (err) {
+    console.error('Invalid date.', jobData.created)
+  }
+
+  function removeHTML(text) {
+    // this regex removes html tags
+    return text.replace(/<[^>]*(>|$)|&nbsp;|&zwnj;|&raquo;|&laquo;|&bull;|&gt;/g, '')
+  }
+
+  function isRemote(jobData) {
+    if (
+      jobData.description.toLowerCase().indexOf('remote') > -1 ||
+      jobData.description.toLowerCase().indexOf('work from home') > -1 ||
+      jobData.title.toLowerCase().indexOf('remote') > -1 ||
+      jobData.title.toLowerCase().indexOf('remote') > -1
+    ) {
+      return ' Yes'
+    }
+
+    return ' No'
+  }
+
+  return (
+    <div className="panel panel-default">
+      <div className="panel-body">
+        <h4>{removeHTML(jobData.title)}</h4>
```

Comment:
Nit: I think it would look nice if all this data was sanitized before it gets down here in the JSX. IMO it makes the JSX look messy.

Reply:
do this in the parent component? Is a sanitizing library an option?

Reply:
U can do it in the [current function body](https://github.com/Vets-Who-Code/vets-who-code-app/pull/278/files#diff-c140b1cd54cc14c0d77d8af6d1b69d4a1e5456147d4379c8fcb3629e651d0392R5-R25). I like to put all my variables at the top. I don't think you need a sanitizing. I think what you're doing is fine and works. My comment is purely about personal style and cleanliness.